avr/cores/megacommand/MidiUart.cpp

```diff
@@ -277,14 +277,12 @@ bool MidiUartClass::avail() { return !rxRb.isEmpty(); }
 uint8_t MidiUartClass::m_getc() { return rxRb.get(); }
 
 ISR(USART1_RX_vect) {
-  uint8_t old_ram_bank = switch_ram_bank(0);
+  select_bank(0);
   isr_midi();
-  switch_ram_bank(old_ram_bank);
```

Comment:
The ISR functions can be called at any time during program execution, this includes during memcpy_bank1()

If we were in BANK 1 before the call to ISR(), then we must switch back to BANK1 when exiting the ISR.